### PR TITLE
[WIP] Vectorizing Guides

### PR DESCRIPTION
**Pull request by jtwinrelevanceai**

### CODE REVIEW

File: guides/vectorizing_guide.ipynb [hunk 1]
```diff
@@ -0,0 +1,3643 @@
+{
```
Comment (boba-and-beer) [inline guides/vectorizing_guide.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    !pip install -U RelevanceAI-dev
</pre><p>Users should not see this. Please remove this output. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/RelevanceAI/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (boba-and-beer) [inline guides/vectorizing_guide.ipynb:1]: <p>REmove this from the guide as well </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/RelevanceAI/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (boba-and-beer) [inline guides/vectorizing_guide.ipynb:1]: <p>Delete this </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/RelevanceAI/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (boba-and-beer) [inline guides/vectorizing_guide.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    dataset.vectorize(feature_vector=True)
</pre><p>FYI a <code>feature_vector</code> does not mean this when it comes to deep neural nets. <code>feature_vector</code>  should be called <code>numeric_vector</code>  IMO. Thoughts? See here: https://www.tensorflow.org/hub/common_signatures/images#:~:text=An%20image%20feature%20vector%20is,learned%20by%20the%20publisher%20model.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/RelevanceAI/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (boba-and-beer) [inline guides/vectorizing_guide.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    !pip install vectorhub[encoders-text-tfhub-windows] # If you are on windows
</pre><p>remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/RelevanceAI/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (jtwinrelevanceai) [reply guides/vectorizing_guide.ipynb:1]: users need to install this to use the default vectorizers

**Pull request closed without merging**